lib/std/os/linux/io_uring.zig

```diff
@@ -636,6 +636,51 @@ pub const IO_Uring = struct {
             @ptrCast(*const c_void, fds.ptr),
             @intCast(u32, fds.len),
         );
+        try convert_registration_error(res);
+    }
+
+    /// Registers the file descriptor for an eventfd that will be notified of completion events on
+    //  an io_uring instance.
```

Comment:
It could probably be useful to note that only one eventfd can be registered on the ring at any given point. Same for `register_eventfd_async`.